src/PatientChecking/PatientChecking/Services/Patient/PatientService.cs

```diff
@@ -75,46 +75,36 @@ public async Task<PatientCheckIn.DataAccess.Models.Patient> GetPatientInDetail(i
             return patient;
         }
 
-        public async Task<int> GetPatientsSummary()
+        public async Task<int> GetPatientsSummaryAsync()
         {
             var NumberOfPatients = await _patientCheckInContext.Patients.ToListAsync();
             return NumberOfPatients.Count();
         }
 
         public async Task<int> UpdatePatientDetail(PatientCheckIn.DataAccess.Models.Patient patientDetails)
         {
-            var patient = _patientCheckInContext.Patients.Find(patientDetails.PatientId);
-
-            patient.FirstName = patientDetails.FirstName;
-            patient.MiddleName = patientDetails.MiddleName;
-            patient.LastName = patientDetails.LastName;
-            patient.FullName = patientDetails.FullName;
-            patient.Gender = patientDetails.Gender;
-            patient.MaritalStatus = patientDetails.MaritalStatus;
-            patient.DoB = patientDetails.DoB;
-            patient.Nationality = patientDetails.Nationality;
-            patient.EthnicRace = patientDetails.EthnicRace;
-            patient.HomeTown = patientDetails.HomeTown;
-            patient.BirthplaceCity = patientDetails.BirthplaceCity;
-            patient.InsuranceNo = patientDetails.InsuranceNo;
-            patient.IdcardNo = patientDetails.IdcardNo;
-            patient.IssuedDate = patientDetails.IssuedDate;
-            patient.IssuedPlace = patientDetails.IssuedPlace;
-
-            _patientCheckInContext.Update(patient);
-
-            return await _patientCheckInContext.SaveChangesAsync();
+            if(patientDetails != null)
+            {
+                _patientCheckInContext.Update(patientDetails);
+
+                return await _patientCheckInContext.SaveChangesAsync();
+            }
+            return -1;
         }
 
         public async Task<int> UploadPatientImage(int patientId, string avatarLink)
```

Comment:
Please append the "Async" suffix for the async method

Reply:
Long has added  "Async" suffix for his the async method created in his PR @harryul